### DISCUSSION TITLE
Request for new connector AppLovin

### DISCUSSION THREAD

**Discussion by jarvislin94** [airbytehq/airbyte#48844] [New Connector Request]
https://developers.applovin.com/en/app-discovery/getting-started

**Comment by tomdarmon-appstack** [comment]
Hey, would be highly interested for this one :)

**Comment by CtrlAltDeploy** [comment]
Our business needs this.

**Comment by sheinbergon** [comment]
To whom it may concern - https://github.com/airbytehq/airbyte/pull/81418/changes
